### PR TITLE
New Plugins

### PR DESCRIPTION
**Pull request by quimodotcom**

### CODE REVIEW

**Comment by rnayabed** [comment]
There are merge conflicts. Please pull the recent changes, update, and then try again

```
.gitignore
README.md
build.sh
obssuite/mother/pom.xml
obssuite/mother/src/main/java/mother/Mother.java
obssuite/mother/src/main/java/mother/OBSActionConnectionTask.java
obssuite/mother/src/main/java/mother/motherconnection/MotherConnection.java
obssuite/setcurrentprofile/pom.xml
obssuite/setcurrentscene/pom.xml
obssuite/setcurrenttransition/pom.xml
obssuite/setmute/pom.xml
obssuite/setmute/src/main/java/module-info.java
obssuite/setmute/src/main/java/setmute/SetMute.java
obssuite/setpreviewscene/pom.xml
obssuite/setrecording/pom.xml
obssuite/setrecording/src/main/java/module-info.java
obssuite/setrecording/src/main/java/setrecording/SetRecording.java
obssuite/setreplaybuffer/pom.xml
obssuite/setstreaming/pom.xml
obssuite/setstreaming/src/main/java/module-info.java
obssuite/setstreaming/src/main/java/setstreaming/SetStreaming.java
obssuite/setstudiomode/pom.xml
obssuite/setstudiomode/src/main/java/SetStudioMode/SetStudioMode.java
obssuite/setstudiomode/src/main/java/module-info.java
obssuite/setvolume/pom.xml
...
```

**Comment by quimodotcom** [comment]
i believe it has been fixed

File: documentopen/.gitignore [hunk 1]
```diff
@@ -1,7 +1,9 @@
 .idea/
```
Comment (rnayabed) [inline documentopen/.gitignore:1]: why was .gitignore file moved from setcurrentprofile to documentopen?

Comment (rnayabed) [reply documentopen/.gitignore:1]: LICENSE from mother (obssuite) was removed and moved to documentopen? why?

File: documentopen/src/main/java/com/stream_pi/documentopen/DocumentOpen.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    public String path = null;
+
+    @Override
+    public void onActionClicked() throws Exception
+    {
+        Property documentloc = getClientProperties().getSingleProperty("documentopen");
+
+        if (documentloc.getStringValue().isBlank())
+        {
+            new StreamPiAlert("Document Open Action", "No file specified", StreamPiAlertType.ERROR).show();
```
Comment (rnayabed) [inline documentopen/src/main/java/com/stream_pi/documentopen/DocumentOpen.java:64]: replace with `throwMinorException("No file specified");`

File: documentopen/src/main/java/com/stream_pi/documentopen/DocumentOpen.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    @Override
+    public void initAction()  {
+
+    }
+
+    public String path = null;
+
+    @Override
+    public void onActionClicked() throws Exception
```
Comment (rnayabed) [inline documentopen/src/main/java/com/stream_pi/documentopen/DocumentOpen.java:58]: `public void onActionClicked() throws MinorException`

**Pull request closed without merging**